src/plot_api/plot_api.js

```diff
@@ -3802,6 +3802,7 @@ function makePlotFramework(gd) {
         .classed('gl-container', true);
 
     fullLayout._paperdiv.selectAll('.main-svg').remove();
+    fullLayout._paperdiv.selectAll('.modebar-container').remove();
```

Comment:
We remove the existing modebar because:
https://github.com/plotly/plotly.js/blob/3e9d162cc4dd9f5b54f3d8e375b46e55fbfb82dc/src/plot_api/plot_api.js#L3792-L3796

Reply:
Done in 567eddedd5ef3e1fd9e45ee0c03a6cea5d279d20